src/main/java/io/github/pylonmc/pylon/base/items/WateringCan.java

```diff
@@ -0,0 +1,173 @@
+package io.github.pylonmc.pylon.base.items;
+
+import com.destroystokyo.paper.ParticleBuilder;
+import io.github.pylonmc.pylon.core.item.PylonItem;
+import io.github.pylonmc.pylon.core.item.PylonItemSchema;
+import io.github.pylonmc.pylon.core.item.base.BlockInteractor;
+import org.bukkit.Material;
+import org.bukkit.Particle;
+import org.bukkit.Sound;
+import org.bukkit.Tag;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.data.Ageable;
+import org.bukkit.entity.Player;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Random;
+
+
+public class WateringCan extends PylonItem<PylonItemSchema> implements BlockInteractor {
+    public static final int RANGE = 4;
+    private static final double CROP_CHANCE = 0.01;
+    private static final double SUGAR_CANE_CHANCE = 0.007;
+    private static final double CACTUSE_CHANCE = 0.01;
```

Comment:
```suggestion
    private static final double CACTUS_CHANCE = 0.01;
```
Do they spell cactus differently across the pond?

Reply:
C A C T U S E